plugin/al/lib/AL_USDMaya/AL/usdmaya/nodes/ProxyShape.cpp

```diff
@@ -2138,24 +2141,20 @@ void ProxyShape::deserialiseTransformRefs()
         {
           MFnDependencyNode fn(node);
           Scope* transformNode = dynamic_cast<Scope*>(fn.userNode());
+          const uint32_t required = tstrs[2].asUnsigned();
+          const uint32_t selected = tstrs[3].asUnsigned();
+          const uint32_t refCounts = tstrs[4].asUnsigned();
+          SdfPath path(tstrs[1].asChar());
+          m_requiredPaths.emplace(path, TransformReference(node, transformNode, required, selected, refCounts));          
           if(transformNode)
```

Comment:
Can you confirm that this is the core change - before you were calling `recordUsdPrimToMayaPath` for everything that WAS NOT a transformNode and now you do it only for transformNode's.

Reply:
Not exactly - recordUsdPrimToMayaPath was not previously being called at all, which is why we got the wrong answer on scene reload. Now it's called for every valid prim/Usdmaya scope combination

Reply:
Right...not sure what I was seeing before. Thx!